packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -151,6 +157,21 @@ const Messages: FunctionComponent<Props> = ({
 
   const [deletedThreads, setDeletedThreads] = useState<string[]>([])
 
+  const getMessageText = (number: number): TranslationMessage => {
```

Comment:
1. Can we change the name `number` to something more meaningful? Maybe `deletedThreadsLength` or something like this?
2. Can we change the name `getMessageText` to something more meaningful? Maybe `getDeletedThreadText` or something like this?

Reply:
Names changed in [7c222ee](https://github.com/mudita/mudita-center/pull/764/commits/7c222eed5a12a682f3d9b7d9ec91c06d8a7edc54)